test/benchmarks/scheduling_passes.py

```diff
@@ -109,14 +109,6 @@ def setup(self, n_qubits, depth):
             dt=1e-9,
         )
         self.timed_dag = TimeUnitConversion(self.durations).run(self.dag)
-        dd_sequence = [XGate(), XGate()]
-        pm = PassManager(
-            [
-                ALAPScheduleAnalysis(self.durations),
-                PadDynamicalDecoupling(self.durations, dd_sequence),
-            ]
-        )
-        self.scheduled_dag = pm.run(self.timed_dag)
```

Comment:
I do not have too much experience running scheduling passes, but I wonder if, to preserve the `time_dynamical_decoupling_pass` test, you could run the analysis pass independently like so: 

```
        pm = PassManager(
            [
                ALAPScheduleAnalysis(self.durations),
            ]
        )
        self.scheduled_dag = pm.run(self.timed_dag)
```
and then use  this`self.scheduled_dag` in `time_dynamical_decoupling_pass`? Maybe you have already tried this and found issues with it.

Reply:
Yes I did try this, but I still received a 'input circuit not scheduled error, please try a scheduling pass', (with or without PadDD in the pass manager) and calling ALAP itself without a pass manager returns None.

Reply:
The thing about that is that these Scheduling Analysis passes, such as `ALAPScheduleAnalysis` and `ASAPScheduleAnalysis`,  need a padding pass to perform the modifications after due to them only modifying the property set of the `PassManager`. 

When we run the sequence within a `PassManager`, `PadDynamicalDecoupling` knows the circuit has been scheduled because of said modification to the PropertySet. We can't really do that outside of a PassManager AFAIK.